packages/typespec-ts/test/modularIntegration/serverEndpointNotDefined.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { assert } from "chai";
+import { NotDefinedClient } from "./generated/server/endpoint/not-defined/src/index.js";
+describe("NotDefined Server Endpoint Client", () => {
+  let client: NotDefinedClient;
+
+  beforeEach(() => {
+    client = new NotDefinedClient("http://localhost:3000", {
+      allowInsecureConnection: true,
+      retryOptions: {
+        maxRetries: 0
```

Comment:
why we need this option?

Reply:
Remove it in https://github.com/Azure/autorest.typescript/pull/2369